docs/source/reference/artifacts.rst

```diff
@@ -4,7 +4,12 @@ optuna.artifacts
 ================
 
 The :mod:`~optuna.artifacts` module provides the way to manage artifacts (output files) in Optuna.
-Please note that methods defined in each ArtifactStore are not intended to be directly accessed by library users.
+
+.. note::
+   The methods defined in each ``ArtifactStore`` are not intended to be directly accessed by library users.
+
+.. note::
+   As ``ArtifactStore`` does not officially provide user API for artifact removal, please refer to `FAQ <how-can-i-delete-all-the-artifacts-uploaded-to-a-study>`_ for the hack.
```

Comment:
[nit] How about adding a link to the artifact tutorial as well?

Reply:
Thank you for the suggestion:) 
i applied your suggestion!